tests/src/admin/tests/utils/mod.rs

```diff
@@ -0,0 +1,143 @@
+use fuels::{
+    prelude::{
+        abigen, launch_custom_provider_and_get_wallets, Contract, LoadConfiguration,
+        StorageConfiguration, TxParameters, WalletUnlocked, WalletsConfig,
+    },
+    programs::call_response::FuelCallResponse,
+    types::Identity,
+};
+
+// Load abi from json
+abigen!(Contract(
+    name = "AdminLib",
+    abi = "src/admin/out/debug/admin_test-abi.json"
+));
+
+pub struct Metadata {
```

Comment:
```suggestion
pub struct User {
```

Reply:
This is not a user as it contains a contract.

Reply:
It belongs to the user though, no?

Either way, `Metadata` is a broad and vague name.